litgpt/api.py

```diff
@@ -474,6 +475,7 @@ def generate(
         Arguments:
             model: The model to use.
             prompt: The prompt string to use for generating the samples.
+            sys_prompt: The system prompt string to use for generating the samples.
```

Comment:
can we pls add what is relation between `prompt` and `sys_prompt` it is become a bit confusing for a user which to use

Reply:
Yes, that sounds perfect